O/openPMD_api/build_tarballs.jl

```diff
@@ -0,0 +1,66 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "openPMD_api"
+version = v"0.13.4"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/openPMD/openPMD-api/archive/refs/tags/0.13.4.tar.gz",
+                  "46c013be5cda670f21969675ce839315d4f5ada0406a6546a91ec3441402cf5e"),
+    DirectorySource("./bundled"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd openPMD-api-0.13.4
+mkdir build
+cd build
+if [[ "$target" == *-apple-* ]]; then
+    # Set up a wrapper script for the assembler. GCC's assembly
+    # output isn't accepted by the LLVM assembler.
+    as=$(which as)
+    mv "$as" "$as.old"
+    export AS="$as.old"
+    ln -s "$WORKSPACE/srcdir/scripts/as.llvm" "$as"
+fi
+mpiopts=
+if [[ "$target" == x86_64-w64-mingw32 ]]; then
+    mpiopts="-DMPI_HOME=$prefix -DMPI_GUESS_LIBRARY_NAME=MSMPI -DMPI_C_LIBRARIES=msmpi64 -DMPI_CXX_LIBRARIES=msmpi64"
+elif [[ "$target" == *-mingw* ]]; then
+    mpiopts="-DMPI_HOME=$prefix -DMPI_GUESS_LIBRARY_NAME=MSMPI"
+fi
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN%.*}_gcc.cmake -DCMAKE_BUILD_TYPE=Release ${mpiopts} ..
+make -j${nproc}
+make -j${nproc} install
+"""
```

Comment:
The license files aren't installed automatically:
```suggestion
install_license ../COPYING*
"""
```